GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Able/RO_Able_Engines.cfg

```diff
@@ -0,0 +1,41 @@
+//	================================================================================
+//	Able Configs
+//	================================================================================
+
+
+// AJ10-37
+@PART[bluedog_Able_Engine]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = AJ10-37
+	@manufacturer = Aerojet
+	@description = Early version of the venerable AJ10 pressure fed upper stage engine using storable propellants. Used on the Able upper stage with the Vanguard and Thor-Able rockets. An upgrade is available for the AJ10-118D as used on Delta B,C,C1 and D.
+	!MODULE[ModuleB9PartSwitch]:HAS[#moduleID[engineSwitch]] {}
+	%engineType = AJ10_Early
+}
+@PART[bluedog_Able_Engine]:AFTER[RealismOverhaulEngines]
+{
+	@title = AJ10-37
+	@manufacturer = Aerojet

```

Comment:
Manufacturer is already set by the `engine_config` (which has localization support), so shouldn't be needed